Bottom: 1dae3f0c743763e0bac6a790ce8518578ccb28e9
Top:    ca42f6e56d8a87f74c7ec776bbeb8af758b80a11
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-26 23:38:01 +0100

Add support for a two soldermask layers, defining regions of removed mask

File-format wise, these work similarly to how silk layers are supported now:

Layer (1 to n "[NAME]")     # "ordinary" design layers
Layer (n+1 "silk")          # top side
Layer (n+2 "silk")          # bottom side
Layer (n+3 "soldermask")    # top side
Layer (n+4 "soldermask")    # bottom side


---

diff --git a/src/const.h b/src/const.h
index 08fe253..2a5a683 100644
--- a/src/const.h
+++ b/src/const.h
@@ -48,6 +48,8 @@
  */
 #define BOTTOM_SILK_LAYER       0
 #define TOP_SILK_LAYER          1
+#define BOTTOM_SOLDERMASK_LAYER 2
+#define TOP_SOLDERMASK_LAYER    3
 
 /* ---------------------------------------------------------------------------
  * misc constants
